src/js/components/awardv2/shared/additionalInfo/AdditionalInfo.jsx

```diff
@@ -29,23 +31,13 @@ export default class AdditionalInfo extends React.Component {
     }
     render() {
         const awardData = this.props.overview;
-        const data = additionalDetails(this.props.overview);
+        const data = this.props.overview._category === 'idv' ? additionalDetailsIdv(this.props.overview) : additionalDetails(this.props.overview);
         // Do not display the Place of Performance section for IDVs
         let placeOfPerformance = null;
-        // Use different Period of Performance and Additional Details mappings for IDVs
         let periodOfPerformance = (
-            <Accordion
-                globalToggle={this.state.globalToggle}
-                accordionName="Period Of Performance"
-                accordionIcon="calendar-alt"
-                accordionData={data.idvPeriodOfPerformance} />
-        );
-        let additionalDetailsSection = (
-            <Accordion
-                globalToggle={this.state.globalToggle}
-                accordionName="Additional Details"
-                accordionIcon="ellipsis-h"
-                accordionData={data.idvAdditionalDetails} />
+            <IdvPeriodOfPerformance
+                accordionData={data.periodOfPerformance}
```

Comment:
I believe we need the globalToggle prop here as well correct?

Reply:
@AGuyNamedMarco noticed this as well, just pushed a fix!